src/client/components/leaderboard/leaderboard.js

```diff
@@ -41,7 +41,7 @@ class Leaderboard extends React.Component {
                 <td>
                   {hasPassed(idx, this) && <div align="center">&#10003;</div>}
                 </td>
-                <td><strong>{this.props.cards[idx]}</strong></td>
+                <td><strong>{this.props.cards[idx] ? this.props.cards[idx] : ''}</strong></td>
```

Comment:
Syntactic sugar:
I believe you could wirte this using the [nullish coalescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator)

```js
<td><strong>{this.props.cards[idx] ?? ''}</strong></td>
```

Reply:
Just tried this, unfortunately it threw this error:
```
SyntaxError: C:\Users\jonesm\Documents\elevation-of-privilege\src\client\components\leaderboard\leaderboard.js: Support for the experimental syntax 'nullishCoalescingOperator' isn't currently enabled (44:52):
```

I can use `||` in this case even if I can't use `??` because cards will never be falsy but not null/undefined. So I think I'll just do this